src/components/Background/index.view.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import "./Background.scss";
+import { ReactComponent as Wire } from "images/wire.svg";
+import { ReactComponent as Rainbow } from "images/rainbowpath.svg";
+interface BackgroundProps {
+  children?: React.ReactNode;
+}
+
+const Background: React.FC<BackgroundProps> = ({
+  children,
+}: BackgroundProps) => {
+  return (
+    <div>
+      <div className="Background">
+        <div className="Background__children">
+          {Array.isArray(children)
+            ? children.filter((child, index) => index < 4)
+            : children}
```

Comment:
There's no need to check to see if `children` prop is an array. We can simply just drop the `children` props in here

Reply:
If I simply drop the children props how would I section the pages?

Reply:
I think "drop" would be the wrong word here. What I was trying to say was that we can simply have this block be: 
```suggestion
          {children}
```